CHANGELOG.md

```diff
@@ -1,42 +1,100 @@
 # Unreleased
+
+Unified spending keys are now used in all places where spending authority
+is required, both for performing spends of shielded funds and for shielding
+transparent funds. Unified spending keys are represented as opaque arrays
+of bytes, and FFI methods are provided to permit derivation of viewing keys
+from the binary unified spending key representation.
+
+IMPORTANT NOTE: the binary representation of a unified spending key may be
+cached, but may become invalid and require re-derivation from seed to use as
```

Comment:
```suggestion
persisted locally, but may become invalid and require re-derivation from seed to use as
```

Caching has a volatile connotation that we might want to avoid conveying

Reply:
That's exactly why I used the word `cached` here: the key should be considered as volatile, and deriving from seed is preferable, because that way when we add new components to the spending key, the caller of this API will not need to implement a separate code path to handle the "your old key was invalidated by the recent upgrade" problem.

Reply:
Oh, yeah, that's right. Also on a side-note caches aren't encrypted either. I think being really clear on the second part of this documentation will solve that.